core/src/main/java/org/bitcoinj/core/Block.java

```diff
@@ -17,21 +17,21 @@
 
 package org.bitcoinj.core;
 
-import com.google.common.annotations.*;
```

Comment:
Please do not reorder imports if the topic of your PR is actually something else.